Tools/Source/libs/AOERandomizer.Logging/Enums/LogLevel.cs

```diff
@@ -3,7 +3,7 @@
     /// <summary>
     /// Enum definition for different log message levels.
     /// </summary>
-    public enum LogLevel
+    internal enum LogLevel
```

Comment:
INTERNAL WHAT IS THIS?????

Reply:
private: accessible only to class
protected: accessible only to derived classes
internal: accessible only to classes in the same assembly
public: accessible to errybuddyyyyyy

Reply:
Since only LogBase uses it right now, it shouldn't be exposed to outside code, essentially (from other assemblies that reference this one).